src/transaction.d.ts

```diff
@@ -1,6 +1,6 @@
-import { Deferrable } from './deferrable';
-import { Logging } from './model';
-import { Sequelize } from './sequelize';
+import { Deferrable } from "./deferrable";
```

Comment:
Can you undo these changes to the quotes? That reduces the amount of changes

Reply:
I have added a prettierrc file and those changes were corrected in subsequent versions. Please let me know if you want me to remove it.